packages/web/src/common/store/search-bar/sagas.ts

```diff
@@ -82,9 +82,6 @@ function* watchSearch() {
     if (lastTask) {
       // cancel is no-op if the task has already terminated
       yield* cancel(lastTask)
-
-      // Reset the search bar state
-      yield* put(searchActions.clearSearch())
```

Comment:
interesting, what was happening here?

Reply:
UX is way better if we don't clear the autocomplete each time you type a char which leads to the history view showing. Same with web (tested there too)